reactioncommerce/reaction
# Dropdown aligns with the button

### Current view
<img width="243" alt="screen shot 2017-11-09 at 5 12 04 pm" src="https://user-images.githubusercontent.com/7762605/32603950-ae0bfe86-c571-11e7-80b2-aecb941ef0be.png">


### View after this fix
<img width="278" alt="screen shot 2017-11-09 at 5 10 00 pm" src="https://user-images.githubusercontent.com/7762605/32603856-4c56ba46-c571-11e7-858b-893530860604.png">

### Work done
Removed the padding and increased the width of button, so that all the parent components adjust their width.

### To test
1. Run `reaction` click on the `Language select` dropdown and `Currency select` dropdown.
2. Note that they have consistent dropdown position now.
